steps/tuist/0.2.0/step.yml

```diff
@@ -0,0 +1,44 @@
+title: tuist
+summary: |
+  Run Tuist from Bitrise's CI environment
+description: |
+  Run Tuist from Bitrise's CI environment
+website: https://github.com/tuist/bitrise-step-tuist
+source_code_url: https://github.com/tuist/bitrise-step-tuist
+support_url: https://github.com/tuist/bitrise-step-tuist/issues
+published_at: 2022-12-24T11:20:38.920791+01:00
+source:
+  git: https://github.com/tuist/bitrise-step-tuist.git
+  commit: 70c07d42370ea9ec6c19a0c1aa5a5f3cb9c1c4a5
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- macos
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
```

Comment:
You are using curl in the step.sh, not git and wget.

Reply:
@danyf90 I was also surprised to see those `wget` and `git` dependencies in the tuist repo yaml while the previous version of the step had `curl` as dependecy.

https://github.com/tuist/bitrise-step-tuist/blob/70c07d42370ea9ec6c19a0c1aa5a5f3cb9c1c4a5/step.yml#L36-L37

I believe those should be changed in the tuist repo as well?

Reply:
Merged and released 0.2.1

Reply:
@koral--  Now the step.yml has the correct dependencies